engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/transformer/UserTaskTransformer.java

```diff
@@ -80,7 +80,8 @@ private void transformAssignmentDefinition(
         // static candidateGroups must be in CSV format, but this is already checked by validator
         jobWorkerProperties.setCandidateGroups(
             ExpressionTransformer.parseListOfCsv(candidateGroups)
-                .map(ExpressionTransformer::asListLiteralExpression)
+                .map(ExpressionTransformer::asListLiteral)
+                .map(ExpressionTransformer::asFeelExpression)
```

Comment:
:x: :question: What I don't understand is the sequence of `asFeelExpression` and `parseExpression`. Isn't one the inverse of the other?  I thought the purpose of this is to transform it into a feel expression.

Reply:
I'm trying to understand your question. `asFeelExpression` does return a FEEL expression (FEEL expressions are prepended with `=` character), but it does so as a String and not as a parsed `Expression`.

Then the question could be: why does it not parse the expression in one step? 🤔 

I guess I have an intent for the ExpressionTransformer to be a utility class for transforming all kinds of expressions that are just represented as strings. I guess I didn't want to add dependencies to `io.camunda.zeebe.el` in this class and keep it simple. Perhaps this method can be reused in other places. Considering that prefixing strings with `=` makes them a feel expression in zeebe is a current design choice, I thought it might get to see some reuse.

Reply:
We should avoid confusions 😄 

I like the suggestion 👍 Thanks